source/boot_security.c

```diff
@@ -1,11 +1,73 @@
+#if !defined(MBEDTLS_CONFIG_FILE)
+#include "mbedtls/config.h"
+#else
+#include MBEDTLS_CONFIG_FILE
+#endif
+#ifndef __cplusplus
+typedef unsigned char bool;
+static const bool False = 0;
+static const bool True = 1;
```

Comment:
Please get rid of above True/False and use true and false definitions when needed below